tests/psalm-baseline.xml

```diff
@@ -5,7 +5,32 @@
       <code><![CDATA[str_starts_with($sK, '_')]]></code>
     </TypeDoesNotContainType>
   </file>
+  <file src="lib/AppInfo/Application.php">
+    <DeprecatedInterface>
+      <code><![CDATA[$context->getAppContainer()]]></code>
```

Comment:
If you add all this to the baseline, you can also simply not enable the checking.
The next PR in server to remove one of these will still break the app and there was no warning visible as a todo list item?